android/src/main/java/io/ably/flutter/plugin/AblyMessageCodec.java

```diff
@@ -325,6 +325,7 @@ private PlatformClientOptions decodeClientOptions(Map<String, Object> jsonMap) {
 
     o.agents = new HashMap<>();
     o.agents.put("ably-flutter", BuildConfig.FLUTTER_PACKAGE_PLUGIN_VERSION);
+    o.agents.put("dart", (String) jsonMap.get(PlatformConstants.TxClientOptions.dartVersion));
```

Comment:
I don't think so - Dart version is passed along with `ClientOptions` so I doubt it can be null here. This is the way all parameters are read on Android side.